test/deprecated/FileSystemPermissions.good

```diff
@@ -1,5 +1,4 @@
 FileSystemPermissions.chpl:1: warning: 'FileSystem.getMode()' is deprecated, please use 'OS.POSIX.stat()' instead
-FileSystemPermissions.chpl:2: warning: 'FileSystem.copyMode()' is deprecated. Please use 'OS.POSIX.stat()' and 'OS.POSIX.chmod()' instead.
```

Comment:
I'm not seeing why this changed—is it obvious to you?

Reply:
Perhaps @dlongnecke-cray has more input, but I think the change is that copyMode is overloaded and the deprecated overload is not used here.

Reply:
Except I see it is called. Hm...

Reply:
Oh, I see. We don't issue deprecation warning on _import_ of overloaded deprecated functions, but only when they are called. Thus, there's still a warning for the use of copyMode on line 8, just not when it's imported.

Reply:
This applies only to overloaded functions in import statements.

Production seems to consider the last overloaded candidate arbitrarily - that didn't make any sense to me, so I opted not to do print anything if a function is overloaded. Another option is to cycle through and print messages for all overloaded candidates.

Reply:
I see, thanks for clarifying.  And I agree that we shouldn't print all of the deprecation messages for an overload.  If all overloads are deprecated (as in this case), we could print one... but which one?  Anyway, this seems OK to me, thanks again.